extensions/dicom-p10-downloader/src/commandsModule.js

```diff
@@ -0,0 +1,47 @@
+import OHIF from '@ohif/core';
+import {
+  getDicomWebClientFromContext,
+  getSOPInstanceReferenceFromActiveViewport,
+} from './utils';
+import downloadAndZip from './downloadAndZip';
+
+export function getCommands(context) {
+  const actions = {
+    downloadAndZipSeriesOnActiveViewport({ servers, viewports, progress }) {
+      const dicomWebClient = getDicomWebClientFromContext(context, servers);
+      if (dicomWebClient) {
```

Comment:
Shouldn't we just do if (!dicomWebClient) throw new Error...

Failing silently doesn't seem good here

Reply:
Great point! I'll remove this conditional as this erroneous condition is properly handled (an exception is thrown) in thee lower layer.